tox.ini

```diff
@@ -13,38 +13,41 @@ envlist =
     coverage
 
 [testenv]
-pip_pre = py313: true
```

Comment:
Maybe this line is the clue to get a dev version of `RestrictedPython`.

Reply:
See above, I don't see a point anymore trying to force this package to find an obscure prerelease. It can wait until RestrictedPython is released.